pytorch_lightning/callbacks/early_stopping.py

```diff
@@ -224,15 +225,34 @@ def _evalute_stopping_criteria(self, current: torch.Tensor) -> Tuple[bool, str]:
             )
         elif self.monitor_op(current - self.min_delta, self.best_score):
             should_stop = False
+            reason = self._improvement_message(current)
             self.best_score = current
             self.wait_count = 0
         else:
             self.wait_count += 1
             if self.wait_count >= self.patience:
                 should_stop = True
                 reason = (
-                    f"Monitored metric {self.monitor} did not improve in the last {self.wait_count} epochs."
+                    f"Monitored metric {self.monitor} did not improve in the last {self.wait_count} records."
                     f" Best score: {self.best_score:.3f}. Signaling Trainer to stop."
                 )
 
         return should_stop, reason
+
+    def _improvement_message(self, current: torch.Tensor) -> str:
+        """ Formats a log message that informs the user about an improvement in the monitored score. """
+        if torch.isfinite(self.best_score):
+            msg = (
+                f"Metric {self.monitor} improved by {abs(self.best_score - current):.3f} >="
+                f" min_delta = {abs(self.min_delta)}. New best score: {current:.3f}"
+            )
+        else:
+            msg = f"Metric {self.monitor} improved. New best score: {current:.3f}"
+        return msg
+
+    @staticmethod
+    def _log_info(trainer: Optional["pl.Trainer"], message: str) -> None:
+        if trainer is not None and trainer.world_size > 1:
+            log.info(f"[{trainer.global_rank}] {message}")
```

Comment:
I'd include global rank here as part of the message too so users know what the number in the bracket refers to 
[global rank:{global_rank}]: ...